js/stores/appStore.js

```diff
@@ -338,11 +338,16 @@ const handleAppAction = (action) => {
       mainWindow.webContents.on('crashed', (e) => {
         console.error('Window crashed. Reloading...')
         mainWindow.loadURL(appUrlUtil.getIndexHTML())
-        ipcMain.once(messages.NOTIFICATION_RESPONSE, (e, message, buttonIndex, persist) => {
+
+        function notificationResponseCallback (e, message, buttonIndex, persist) {
           if (message === locale.translation('unexpectedErrorWindowReload')) {
             appActions.hideMessageBox(message)
+          } else {
+            ipcMain.once(messages.NOTIFICATION_RESPONSE, notificationResponseCallback)
           }
-        })
+        }
+        ipcMain.once(messages.NOTIFICATION_RESPONSE, notificationResponseCallback)
```

Comment:
ditto! opened a PR to switch to ipcMain.removeListener and it is much cleaner now. thanks :-)